### PR TITLE
WL-4017 Fixes for Lessons Forum widget from WL10 is applied to WL11.

### PR DESCRIPTION
**Pull request by ouit0408**
Also, changes from 'msgcntr' module for WL-4017 are added in this commit.
Adjusted code in Lessons to work with SAKAI-11 like edit options for each
item on Lessons page.

### CODE REVIEW

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowItemProducer.java [hunk 1]
```diff
@@ -351,9 +351,20 @@ else if (item.isPrerequisite() && (type == SimplePageItem.PAGE || type == Simple
 	    // this isn't a security issue, since source is /access/lessonbuilder, and
 	    // that will be checked. THat's not the case when the URL is directly present.
 	    // in that case we have to get it from the item.
-	    String source = params.getSource();
-	    if (item != null && item.getAttribute("multimediaUrl") != null)
-		source = item.getAttribute("multimediaUrl");
+		//check if item is of type Forum_Summary
```
Comment (buckett) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowItemProducer.java]: Indentation here looks wrong.

**Comment by ouit0408** [comment]
corrected indentation

**Comment by buckett** [comment]
Sorry, doesn't merge now.

**Comment by ouit0408** [comment]
rebased branch

**Comment by buckett** [comment]
Thanks, 🍰

**Pull request merged**